game-app/game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -742,6 +742,20 @@ private void addZoomKeyboardShortcuts() {
         () ->
             mapPanel.setScale(
                 mapPanel.getScale() - (ClientSetting.mapZoomFactor.getValueOrThrow() / 100f)));
+
+    SwingKeyBinding.addKeyBindingWithMetaAndCtrlMasks(
```

Comment:
Rather than duplicating the above block, what do you think updating  "addKeyBindingWithMetaAndCtrlMasks" to accept a var-arg of key codes?